rust/private/rustfmt.bzl

```diff
@@ -61,7 +63,7 @@ def _perform_check(edition, srcs, ctx):
         executable = ctx.executable._process_wrapper,
         inputs = srcs,
         outputs = [marker],
-        tools = [toolchain.rustfmt],
+        tools = [toolchain.rustfmt, config],
```

Comment:
This feels like more of an `input` than a `tool`? But don't think it really matters which one it's in tbh

Reply:
Updated